lib/faraday/adapter/em_http.rb

```diff
@@ -166,17 +166,17 @@ def error_message(client)
       end
 
       def raise_error(msg)
-        errklass = Faraday::ClientError
-        if msg == Errno::ETIMEDOUT
-          errklass = Faraday::TimeoutError
+        error_class = Faraday::ClientError
+        if msg == Errno::ETIMEDOUT || (msg.is_a?(String) && msg.include?('timeout error'))
```

Comment:
Why was that extra `'timeout error'` check removed?

Reply:
Not sure I follow. I just added an extra check on `msg`, there was no removal